### PR TITLE
1.1.0 Release

### PR DESCRIPTION
**Pull request by TyIsI**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
-  },
-  "cmake-js": {
-    "runtime": "node",
-    "runtimeVersion": "0.12.0",
-    "arch": "x64"
-  }
+    "name": "@vhs/node-nfc-nci",
+    "version": "1.1.0",
+    "description": "NXP NFC NCI Node wrapper",
+    "repository": "https://github.com/vhs/node-nfc-nci",
```
Comment (laftho) [inline package.json:5]: should this be forked over to the vhs org?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
Comment (laftho) [inline .editorconfig:8]: any reason this isn't `indent_size = 2` ?

File: package.json (hunk 1 above)
Comment (laftho) [reply package.json:5]: yup, nvm I see that it is :+1:

File: .editorconfig (hunk 2 above)
Comment (TyIsI) [reply .editorconfig:8]: Whoops! Thought I had opened the PR in the VHS org.

And, nah, it's more my default for readability.

I just released 1.2.0 on https://github.com/vhs/node-nfc-nci

**Comment by TyIsI** [comment]
Closing this one!

**Pull request closed without merging**